packages/react-compose/src/next/simplifyShorthand.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { GenericDictionary } from './types';
+
+/**
+ * Ensures that the given slots are represented using object syntax. This ensures that
+ * the object can be merged along with other objects.
+ * @param props - The incoming props
+ * @param shorthandPropNames - An array of prop names to apply simplification to
+ */
+export const simplifyShorthand = (props: GenericDictionary, shorthandPropNames?: string[]) => {
```

Comment:
I suggest to improve typings: `shorthandPropNames` should be present in `props`

Reply:
I'm a little confused. The array of keys within props are not a prop in props.

Reply:
good suggestion!

Reply:
updated typings.